src/librustc/hir/print.rs

```diff
@@ -735,6 +735,27 @@ impl<'a> State<'a> {
                 }
                 self.bclose(item.span)?;
             }
+            hir::ItemTraitAlias(ref generics, ref bounds) => {
+                self.head("")?;
+                self.print_visibility(&item.vis)?;
+                self.word_nbsp("trait")?;
+                self.print_name(item.name)?;
+                self.print_generics(generics)?;
+                let mut real_bounds = Vec::with_capacity(bounds.len());
+                // FIXME(durka) this seems to be some quite outdated syntax
```

Comment:
yeah, you used to have to say `trait Foo for ?Sized`, but now that's the default (and you have to write `trait Foo: Sized`)

Reply:
I would assume that `TraitBoundModifier::Maybe` in this context is illegal, right? i.e., `trait Foo = ?Sized` is kind of nonsense.

Reply:
Agreed.